test/dynamo/test_dynamic_shapes.py

```diff
@@ -43,26 +43,22 @@
 XFAIL_HITS = 0
 
 
-def make_dynamic_cls(cls, *, static_default=False, unspec=False):
+def make_dynamic_cls(cls, *, static_default=False):
     suffix = "_dynamic_shapes"
     if static_default:
         suffix += "_static_default"
-    if unspec:
-        suffix += "_unspec"
 
     cls_prefix = "DynamicShapes"
     if static_default:
         cls_prefix = f"StaticDefault{cls_prefix}"
-    if unspec:
-        cls_prefix = f"Unspec{cls_prefix}"
 
     test_class = make_test_cls_with_patches(
         cls,
         cls_prefix,
         suffix,
         (config, "dynamic_shapes", True),
         (config, "assume_static_by_default", static_default),
-        (config, "specialize_int", not unspec),
+        (config, "specialize_int", static_default),
```

Comment:
Question: This code pairs assume_static_by_default with specialize_int; both are True or both are False (the default). Is it also conceivable to have one be True while the other False? The config in line 86 _used to be_ assume_static_by_default = False and specialize_int = True and now means both = False.

Reply:
Yes, I have reduced coverage. This is a calculated decision: assume static by default is heavily correlated with specializing ints, as these are correlated with export usage. In eager usage, we expect unspec int and not assuming static by default. There's two configs that are not tested, but I think testing them is unnecessary as we will get adequate coverage otherwise.